src/modules/commentCounter.js

```diff
@@ -0,0 +1,12 @@
+const commentCounter = async (mealId) => {
+  const response = await fetch(
+    `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/o5YLWfucVuUSY3jbCcrI/comments?item_id=${mealId}`,
+  );
```

Comment:
As part of the project requirements, confidential information such as the app_id (like `o5YLWfucVuUSY3jbCcrI` ) should not be stored in Github. Kindly consider storing them in something like a .env file or similar and sharing them with the code reviewer on the pull request description instead of pushing them to Github. :wink: . For further instructions on how to create a .env file, please check [this link](https://www.twilio.com/blog/how-to-set-environment-variables-html)